addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -185,6 +185,14 @@
     </t>
     <t t-name="wysiwyg.widgets.files.existing.content">
         <div class="existing-attachments">
+            <div t-if="rows.length === 0" class="o_nocontent_help">
+                <p class="o_empty_folder_image">
+                    <t t-if="isDocument">No documents found</t>
+                    <t t-else="">No images found</t>
+                </p><p>
+                    You can upload <t t-if="isDocument">documents</t><t t-else="">images</t> with the button located in the top left of the screen
```

Comment:
This is not translatable. Use string formatting or duplicate the string for both type of files.